web/src/routes/(user)/utilities/duplicates/[[photos=photos]]/[[assetId=id]]/+page.ts

```diff
@@ -1,13 +1,33 @@
 import { authenticate } from '$lib/utils/auth';
 import { getFormatter } from '$lib/utils/i18n';
 import { getAssetInfoFromParam } from '$lib/utils/navigation';
-import { getAssetDuplicates } from '@immich/sdk';
+import { getAssetDuplicates, updateAssets, type DuplicateResponseDto } from '@immich/sdk';
 import type { PageLoad } from './$types';
 
+const rectifyDuplicate = (uniqueAssetIds: string[]) => {
+  if (uniqueAssetIds.length > 0) {
+    void updateAssets({ assetBulkUpdateDto: { ids: uniqueAssetIds, duplicateId: null } });
+  }
+};
+
+const processDuplicates = (duplicates: DuplicateResponseDto[]) => {
+  const uniqueAssetIds: string[] = [];
+  const validDuplicates = duplicates.filter((duplicate) => {
+    if (duplicate.assets.length <= 1) {
+      uniqueAssetIds.push(duplicate.assets[0].id);
+      return false;
+    }
+    return true;
+  });
+  rectifyDuplicate(uniqueAssetIds);
+  return validDuplicates;
+};
+
 export const load = (async ({ params }) => {
   await authenticate();
   const asset = await getAssetInfoFromParam(params);
-  const duplicates = await getAssetDuplicates();
+  const Allduplicates = await getAssetDuplicates();
```

Comment:
This should be camel-case. Also, while you're changing this, could you also put these promises (except `authenticate`) in a `Promise.all` call so they happen in parallel?